usr/sbin/laptop_mode

```diff
@@ -456,8 +456,9 @@ lmt_load_config ()
 	BATTERY_NOT_DISCHARGING=0
 	for POWER_SUPPLY in /sys/class/power_supply/* ; do
 		if [ -f $POWER_SUPPLY/type ] ; then
-			FOUND_SYS_CLASS_POWER_SUPPLY_AC=1
```

Comment:
This is going to stay. As I explained in the email this is just the determine the power system type